CMakeLists.txt

```diff
@@ -120,6 +120,9 @@ endif()
 # Installation
 #
 install(TARGETS ccache DESTINATION ${CMAKE_INSTALL_BINDIR})
+if(WIN32)
+  install(FILES ${CMAKE_CURRENT_LIST_DIR}/ccache.exe.manifest TYPE BIN DESTINATION ${CMAKE_INSTALL_BINDIR})
```

Comment:
I would prefer to put the manifest in some subdirectory to avoid adding more files to the top directory. Perhaps in `misc`?

Reply:
That makes sense, done :)